src/VisualStudio/Core/Impl/Options/Style/NamingPreferences/NamingStyleOptionPageViewModel.cs

```diff
@@ -275,6 +275,34 @@ public bool CanMoveDown
                 }
             }
 
+            public string MoveUpAutomationText
```

Comment:
I'm likely missing something, but I don't see where this property is consumed, similar with `MoveDownAutomationText`.